fmriprep/config.py

```diff
@@ -573,18 +573,19 @@ def init(cls):
 class seeds(_Config):
     """Initialize the PRNG and track random seed assignments"""
 
+    _random_seed = None
     master = None
-    """Master seed used to generate all other tracked seeds"""
+    """Master random seed to initialize the Pseudorandom Number Generator (PRNG)"""
     ants = None
     """Seed used for antsRegistration, antsAI, antsMotionCorr"""
 
     @classmethod
     def init(cls):
-        cls.master = workflow.random_seed
+        if cls._random_seed is not None:
+            cls.master = cls._random_seed
         if cls.master is None:
             cls.master = random.randint(1, 65536)
         random.seed(cls.master)  # initialize the PRNG
-
         # functions to set program specific seeds
         cls.ants = _set_ants_seed()
```

Comment:
This should depend on the master seed, so that it can be reproduced by setting the master seed.

Reply:
isn't that already the case? the seed is set above, and then the method is called.

Reply:
Oh, right. My eyes skipped over that.